src/factories/request.ts

```diff
@@ -32,6 +33,8 @@ export default class Request {
     this.authenticatedInstance.interceptors.request.use(
       this.createAuthInterceptor,
     );
+
+    this.customAxiosConfig = {}; // Initialize with an empty object
   }
 
   setInstanceConfig = (configEnv: Env): void => {
```

Comment:
we only need one instance of Axios, so you will need to pass a custom config as a second parameter here.

Reply:
20125dfc8315b789747e3a27465bdcd929cd8101